### PR TITLE
Library books/ bookbinder

### PR DESCRIPTION
**Pull request by Xxxxyobdab**
- Leather sheets can be used to reskin books
- The bookbinder can now be set to print a book with a specific cover
- Books now retain reference to it's category
- books spawned from only the library bookspawners will gain the category of it.
- books printed from the external archive will fill in the category var
- the barcode scanner's base function will now also tell title, author, category on scan.

Please ensure the code used for database query is correct as I cannot test for it.

### CODE REVIEW

**Comment by Xxxxyobdab** [comment]
!review required

**Review by Alberyk — CHANGES_REQUESTED** [review]
Do not delete the changelos or replace one with yours. Create a new file for your changelog.

File: code/modules/library/lib_machines.dm [hunk 1]
```diff
@@ -365,18 +365,21 @@ datum/borrowbook // Datum used to keep track of who has borrowed what when and f
 			bibledelay = 1
 			spawn(60)
 				bibledelay = 0
-			var/DBQuery/query = dbcon.NewQuery("SELECT * FROM ss13_library WHERE id=[sqlid]")
+			//var/DBQuery/query = dbcon.NewQuery("SELECT * FROM ss13_library WHERE id=[sqlid]")
+			var/DBQuery/query = dbcon.NewQuery("SELECT author, title, content, category FROM ss13_library WHERE id=[sqlid]")
 			query.Execute()
```
Comment (Arrow768) [inline code/modules/library/lib_machines.dm:370]:
```suggestion
			var/DBQuery/query = dbcon.NewQuery("SELECT author, title, content, category FROM ss13_library WHERE id=:sqlid:")
			query.Execute(list("sqlid"=sqlid))
```

**Review by Arrow768 — CHANGES_REQUESTED** [review]

**Comment by NonQueueingMatt** [comment]
Inactive, DM me if you want this reopened.

**Pull request closed without merging**